spec/dummy/app/avo/resources/test_resource.rb

```diff
@@ -0,0 +1,11 @@
+class TestResource < Avo::BaseResource
```

Comment:
Can we change the name to something more expressive like `TinyComment` (I'm open to something else)?

Reply:
changed to PhotoComment because now have a photo attached